packages/components/src/mobile/audio-player/index.native.js

```diff
@@ -131,42 +137,61 @@ function Player( {
 		);
 	};
 
+	function onAudioUploadCancelDialog() {
+		if ( isUploadInProgress ) {
+			requestImageUploadCancelDialog( attributes.id );
+		} else if (
+			attributes.id &&
+			getProtocol( attributes.src ) === 'file:'
+		) {
+			requestImageFailedRetryDialog( attributes.id );
+		}
+	}
+
 	return (
-		<View style={ containerStyle }>
-			<View style={ iconContainerStyle }>
-				<Icon icon={ audio } style={ finalIconStyle } size={ 24 } />
-			</View>
-			<View style={ titleContainerStyle }>
-				<Text style={ titleStyle }>{ title }</Text>
-				<View style={ styles.subtitleContainer }>
-					{ isUploadFailed && (
-						<Icon
-							icon={ warning }
-							style={ {
-								...styles.errorIcon,
-								...uploadFailedStyle,
-							} }
-							size={ 16 }
-						/>
-					) }
-					<Text style={ finalSubtitleStyle }>
-						{ getSubtitleValue() }
-					</Text>
+		<TouchableWithoutFeedback
+			accessible={ ! isSelected }
```

Comment:
@ceyhun with the `TouchableWithoutFeedback` controls I see where sometimes it only has one prop set and that is `onPress` and other times like here, the accessible and disabled props are passed. Is the pattern for doing this based on how deep in the hierarchy a touch-based component is located? Or is it simpler than that 😅 Let me know :)

Reply:
I think it's related to how deep, nested are the touchables are. In this specific case when we press on the Audio Block we want first the block to be selected and than the second press on the block should trigger the retry dialog.